src/main/java/org/mule/extension/http/api/HttpRequestAttributes.java

```diff
@@ -93,7 +93,7 @@
    */
   @Parameter
   @Optional
-  private transient Certificate clientCertificate;
+  private final Certificate clientCertificate = null;
```

Comment:
you can't do this. The SDK will try to set this value and DW might as well if this is used in a transformation. What you have to do is to store the result of the lazy supplier in this variable, making sure that when the supplier is resolved, the value is always available here. 

This also needs to have a getter which actually uses the field for DW compatibility

Reply:
This is how this was before all these changes. 
Besides, all parameters are declared final, why would they be changed by de SDK and DW? and how?
Why won't this be compatible with DW if it already has a getter that returns the value from the "lazy certificate"? I know that DW uses reflection when no getter is found for the attribute is field is trying to obtain, but if there is a getter present , it makes use of it.